azure-quantum/azure/quantum/target/microsoft/elements/dft/job.py

```diff
@@ -0,0 +1,46 @@
+import collections.abc
+from typing import Any, Dict, Union
+from azure.quantum.job import JobFailedWithResultsError
+from azure.quantum.job.job import Job, DEFAULT_TIMEOUT
+from azure.quantum._client.models import JobDetails
+
+class MicrosoftElementsDftJob(Job):
+    """
+    A dedicated job class for jobs from the microsoft.dft target.
+    """
+
+    def __init__(self, workspace, job_details: JobDetails, **kwargs):
+        super().__init__(workspace, job_details, **kwargs)
+
+
+    def get_results(self, timeout_secs: float = DEFAULT_TIMEOUT) -> Dict[str, Any]:
+        try:
+            job_results = super().get_results(timeout_secs)
+            return job_results["results"]
+        except JobFailedWithResultsError as e:
+                failure_results = e.get_failure_results()
+                if MicrosoftElementsDftJob._is_dft_failure_results(failure_results):
+                    error = failure_results["results"][0]["error"]
+                    message = f'{e.get_message()} Error type: {error["error_type"]}. Message: {error["error_message"]}'
```

Comment:
Does e.get_message() add a dot (.) at the end?
Just making sure the error message looks well formatted

Reply:
The original exception message ends with period, so we don't need to add it here.